popup.css

```diff
@@ -0,0 +1,141 @@
+/* General */
+#main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: 'Roboto', sans-serif;
+}
+
+#num-display {
+  text-align: center;
+  font-size: 1.2em;
+}
+
+#num-display p {
+  font-size: 1.5em;
+}
+
+#settings-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em;
+  border: 1px solid #d3d3d3;
+  user-select: none;
+}
+
+#on-switch-container {
+  margin-top: 1em;
+}
+
+/* Prevent interaction with settings while running */
+.disabled {
+  pointer-events: none;
+}
+
+/* Unit Switch Button */
+
+.unit-switch {
+  display: flex;
+  overflow: hidden;
+  margin: 1em 0 1em 0;
+}
+
+.unit-switch input {
+	position: absolute !important;
+	clip: rect(0, 0, 0, 0);
+	height: 1px;
+	width: 1px;
+  border: 0;
+}
+
+.unit-switch label {
+	background-color: #ffffff;
+	color: rgba(0, 0, 0, 0.6);
+  font-size: 1em;
+	text-align: center;
+	padding: 8px 16px;
+	border: 1px solid rgba(0, 0, 0, 0.2);
+  transition: all 0.1s ease-in-out;
+  
+}
+
+.unit-switch input:checked + label {
+  background-color: #2196F3;
+  color: white;
+	box-shadow: none;
+}
+
+.unit-switch label:first-of-type {
+	border-radius: 4px 0 0 4px;
+}
+
+.unit-switch label:last-of-type {
+	border-radius: 0 4px 4px 0;
+}
+
+/* On/Off Switch */
+
+.on-switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+.on-switch input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.toggle {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  transition: .4s;
+  border-radius: 35px;
+}
+
+.toggle:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
```

Comment:
An observation here, when I open the UI of the extension, the switch slowly moves to right, the expected behaviour should have been that if it is active, the switch moves instantaneously to prevent the visual transition.